### PR TITLE
Nemo#2021/remove check circle

### PR DESCRIPTION
**Pull request by ShepleySound**
## Ticket

[Airtable Ticket #2021](https://airtable.com/appfJZShN8K4tcWHU/tblXQZfKPAJIjV4cL/viwcmDaRCUfJOVZ4a/recGAYbQOEQPRlUZz?blocks=hide)

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)

## Description

I have added some theming rules to override the default styling of the MUI Button component. Now, instead of reaching for the "RedesignPrimaryButton"/"RedesignSecondaryButton" components, developers should be able to use the MUI Button component with all of its expected props. This includes using the underlying MUI color palette, like "error", "success", and "info"! I've explicitly defined 3 themes for our most common use cases using the built-in `variant` and `color` props.

### `variant: "contained"` `color: "primary"`
image

### `variant: "contained"` `color: "secondary"`
image
### `variant: "outlined"` `color: "primary"`
image

### Before
image
image

### After
image
image

## Checklist:

-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation (if applicable)
-   [ ] My changes generate no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
-   [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/Components/Navigation.jsx [hunk 1]
```diff
@@ -96,9 +84,9 @@ const Navigation = () => {
                                     page.key !== PageId.AccessCalculator ? <NavButton key={idx} page={page} /> : null
                                 )}
                             </ButtonGroup>
-                            <RedesignButtonPrimary href="/calculator/landing-0" className={classes.navCalc}>
+                            <Button variant="contained" color="secondary" href="/calculator/landing-0">
```
Comment (stuttskl) [inline src/Components/Navigation.jsx:87]: I love this! Thanks for adding code to help make our UI components a bit more modular :)

File: src/Components/Onboarding/CompleteButton.jsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import React, { useState, useEffect } from "react";
 import { Link, useLocation } from "react-router-dom";
-import { Typography } from "@mui/material";
+import { Typography, Button } from "@mui/material";
 import { useButtonStyles } from "../../Styles/Onboarding/useButtonStyles";
-import openCircle from "../../Assets/Onboarding/openCircle.png";
+// import openCircle from "../../Assets/Onboarding/openCircle.png";
```
Comment (stuttskl) [inline src/Components/Onboarding/CompleteButton.jsx:5]: If we aren't using this, we can just remove the import

File: src/Components/Onboarding/CompleteButton.jsx [hunk 3]
```diff
@@ -46,12 +46,16 @@ const CompleteButton = () => {
...
-            <div className={classes.buttonSecondary} style={isComplete ? styles.success : styles.div}>
-                <img src={isComplete ? circleCheck : openCircle} alt="" style={styles.img} />
-                <Typography variant="button">{isComplete ? "completed" : "mark as done"}</Typography>
-            </div>
-        </Link>
+        <>
+            <Button
+                variant="outlined"
+                color={isComplete ? "success" : "primary"}
+                startIcon={isComplete ? <CheckCircleOutlineIcon /> : null}
```
Comment (stuttskl) [inline src/Components/Onboarding/CompleteButton.jsx:53]: awesome!

**Review by stuttskl — APPROVED** [review]
Approve pending the removal of a line of commented out code. Looks good : )

File: src/Components/ImageContentCard.jsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { Paper, Typography, Box } from "@mui/material";
 import { makeStyles } from "@mui/styles";
 import { RedesignButtonPrimary } from "../ui-kit/RedesignButtonPrimary";
+import { RedesignButtonSecondary } from "../ui-kit/RedesignButtonSecondary";
```
[the remark this answers is not in the document]
Comment (ShepleySound) [reply src/Components/ImageContentCard.jsx:4]: Fixing these now. The terminal spits out the actual linter error, and these seem to trigger both the "no-unused-imports" rule and the "no-unused-var" rule.

**Comment by ShepleySound** [comment]
Got rid of the linter errors by removing the unused component imports!

**Pull request merged**